results/TIGER-Lab__VLM2Vec-Full/e9afa98002097ac2471827ba23ea1f2ddd229480/VOC2007.json

```diff
@@ -0,0 +1,48 @@
+{
+  "dataset_revision": "dbafdb9e1506c9c419c5c4672e409463cd21ba50",
+  "evaluation_time": 1066.543753862381,
+  "kg_co2_emissions": null,
+  "mteb_version": "1.16.5",
+  "scores": {
+    "test": [
+      {
+        "accuracy": 0.5936187399030695,
+        "f1": 0.719121663978135,
+        "hf_subset": "default",
+        "languages": [
+          "eng-Latn"
+        ],
+        "lrap": 0.7205710375157255,
```

Comment:
Using commits to track lrap scores for both models at different `samples_per_label`. Will remove before merging.